### PR TITLE
Nikita Petrov hw1 

### PR DESCRIPTION
**Pull request by Da1nwood**

### CODE REVIEW

File: homework1/Nikita.Petrov/src/main/java/Content.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    private String branchName;
+    private ArrayList<Documents> arrayList = new ArrayList<Documents>();
+
+
+    public Content(String branchName) {
+        this.branchName = branchName;
+    }
+
+
+    public ArrayList<Documents> getArrayList() { return arrayList; }
```
Comment (Dmitry404) [inline homework1/Nikita.Petrov/src/main/java/Content.java]: Don't expose implementation details (ArrayList), use List instead + rename the method.

File: homework1/Nikita.Petrov/src/main/java/Root.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+
+public class Root {
+    private LinkedHashMap<String, Content> branches = new LinkedHashMap<String, Content>();
+    private HashMap<String, Object> cloneHashMap = new HashMap<String, Object>();
```
Comment (Dmitry404) [inline homework1/Nikita.Petrov/src/main/java/Root.java]: - Both fields above should be `Map` 
- Use Diamond operator in the right side

Something like this
```java
 Map<String, ?> cloneHashMap = new HashMap<>();
```

**Comment by Da1nwood** [comment]
Я переделал структуру

**Comment by Dmitry404** [comment]
> Я переделал структуру

Please make the changes we discussed. Make sure your folder's name is in lower case and dot separated - `nikita.petrov`

**Comment by Dmitry404** [comment]
Closing this PR as there is no feedback for two weeks. Please post a new PR once you have the changes ready.

**Pull request closed without merging**